.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# [Choice] Java version: 11, 15
+ARG VARIANT=11
+FROM mcr.microsoft.com/vscode/devcontainers/java:${VARIANT}
+
+# [Option] Install Maven
+ARG INSTALL_MAVEN="false"
+ARG MAVEN_VERSION=""
+# [Option] Install Gradle
+ARG INSTALL_GRADLE="false"
+ARG GRADLE_VERSION=""
+RUN if [ "${INSTALL_MAVEN}" = "true" ]; then su vscode -c "source /usr/local/sdkman/bin/sdkman-init.sh && sdk install maven \"${MAVEN_VERSION}\""; fi \
+    && if [ "${INSTALL_GRADLE}" = "true" ]; then su vscode -c "source /usr/local/sdkman/bin/sdkman-init.sh && sdk install gradle \"${GRADLE_VERSION}\""; fi
+
+# [Option] Install Node.js
```

Comment:
I would remove the `Node` related stuff since in context of hawkBit development it won't do you any good. Same for `Gradle`. I'm afraid that if people see this option they might get the impression that hawkBit could be build with Gradle (which is not the case).

Reply:
removing both options and promote `Maven` as default will simply the setup a lot I think

Reply:
Yeah, good point. Will do. Also over at hawkBit.